project/deps.sc

```diff
@@ -167,8 +167,10 @@ object Deps {
   def shapeless                = ivy"com.chuusai::shapeless:2.3.9"
   def signingCliShared =
     ivy"io.github.alexarchambault.scala-cli.signing::shared:${Versions.signingCli}"
+      .exclude(("com.github.alexarchambault", "case-app_3"))
   def signingCli = ivy"io.github.alexarchambault.scala-cli.signing::cli:${Versions.signingCli}"
-  def slf4jNop   = ivy"org.slf4j:slf4j-nop:2.0.6"
+    .exclude(("com.github.alexarchambault", "case-app_3"))
```

Comment:
Can you add comment, why you exclude this dependencies?

Reply:
sure